src/lib.rs

```diff
@@ -1,8 +1,17 @@
 #![no_std]
+#![feature(const_raw_ptr_deref)]
+#![feature(const_fn)]
```

Comment:
```suggestion
```

These don't seem necessary anymore? I removed them locally, and it still compiles.

Reply:
You're right. I removed them.